tools/manifest/item.py

```diff
@@ -31,14 +22,23 @@ class ManifestItemMeta(ABCMeta):
     attribute, and otherwise behaves like an ABCMeta."""
 
     def __new__(cls, name, bases, attrs):
-        # type: (Type[ManifestItemMeta], str, Tuple[ManifestItemMeta, ...], Dict[str, Any]) -> ManifestItemMeta
-        rv = super(ManifestItemMeta, cls).__new__(cls, name, bases, attrs)
-        if not isabstract(rv):
-            assert issubclass(rv, ManifestItem)
-            assert isinstance(rv.item_type, str)
-            item_types[rv.item_type] = rv
-
-        return rv  # type: ignore
+        # type: (Type[ManifestItemMeta], str, Tuple[type], Dict[str, Any]) -> ManifestItemMeta
+        inst = super(ManifestItemMeta, cls).__new__(cls, name, bases, attrs)
```

Comment:
This is the most complicated change made here. The original errors were:
```
tools/manifest/item.py:38: error: "ManifestItemMeta" has no attribute "item_type"  [attr-defined]
tools/manifest/item.py:39: error: "ManifestItemMeta" has no attribute "item_type"  [attr-defined]
tools/manifest/item.py:39: error: Incompatible types in assignment (expression has type "ManifestItemMeta", target has type "Type[ManifestItem]")  [assignment]
tools/manifest/item.py:41: error: Unused "type: ignore" comment
```

The fix seems fine to me, I don't know that it could be simplified other than by silencing some errors.